readme.md

```diff
@@ -246,7 +246,14 @@ Default: `'default'`
 
 **Note:** When using streams, this option is ignored.
 
-Parsing method used to retrieve the body from the response. Can be `'default'`, `'text'`, `'json'` or `'buffer'`. The promise has `.json()` and `.buffer()` and `.text()` functions which set this option automatically.
+Parsing method used to retrieve the body from the response.
+
+- `'default'` - if `options.encoding` is `null`, the body will be a Buffer. Otherwise it will be a string unless it's overwritten in a `afterResponse` hook,
```

Comment:
Should we remove the `null` thing to get `buffer` as the user can just do `responseType: 'buffer'` instead? Which is also much clearer.

Reply:
Sure, I'll send a PR.